spec/services/vendor_api/generate_test_applications_for_provider_spec.rb

```diff
@@ -1,70 +1,139 @@
 require 'rails_helper'
 
 RSpec.describe VendorAPI::GenerateTestApplicationsForProvider, sidekiq: true do
+  let(:provider) { create(:provider) }
+  let(:courses_per_application) { 3 }
+  let(:application_count) { 1 }
+  let(:for_training_courses) { false }
+  let(:for_ratified_courses) { false }
+  let(:for_test_provider_courses) { false }
+  let(:service_params) do
+    {
+      provider: provider,
+      courses_per_application: courses_per_application,
+      count: application_count,
+      for_training_courses: for_training_courses,
+      for_ratified_courses: for_ratified_courses,
+      for_test_provider_courses: for_test_provider_courses,
+    }
+  end
+
+  before do
+    create(:course_option)
+    # rubocop:disable FactoryBot/CreateList
```

Comment:
why did we need to disable rubocop and not use `create_list`?

Reply:
Because I'm creating a new course each time too. If I use a create_list, then it makes 3 course options all with the same course